### ISSUE TITLE
sway crashing on running slurp and then moving workspace before selecting an area

### ISSUE THREAD

**Issue by biwhite** [swaywm/sway#3795]
sway version 1.0-rc1-118-g9032be4f (Feb 27 2019, branch 'master')

Sway debug log
https://gist.github.com/biwhite/c4a3bd2175036a5ea7e985806066abd0

Backtrace
https://gist.github.com/biwhite/30c70405f417ab302bde25f805bb8a51

Finally, explain the steps you took in plain English to reproduce the problem below.

Crashes when opening sway, running slurp, and using a workspace prev_on_output, either through key bindings or swaymsg.

**Comment by Emantor** [comment]
Investigating the code, the fix is going to be in `seat_get_focused_workspace` by also trying `seat_get_focus_inactive`. However a seat not having a focus or inactive focus behaviour is still valid (e.g. shortly after creation) and should imo be handled by all callers. In this case I think its also fine to abort in the case of the above command.
The other call sites of `seat_get_focused_workspace` need investigation as well.

**Comment by emersion** [comment]
>the fix is going to be in `seat_get_focused_workspace` by also trying `seat_get_focus_inactive`

We should probably only do this if a layer surface is focused?
